content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,625 @@
+  Options are :guilabel:`Day`, if time off can only be requested in full day increments (8 hours),
+  :guilabel:`Half Day`, if time off can only be requested in half day increments (4 hours), or
+  :guilabel:`Hours`, if the time off can be taken in hourly increments.
+- :guilabel:`Deduct Extra Hours`: if the time off request should factor in any extra time accrued by
+  the employee, enable this box. For example, if an employee worked 2 extra hours for the week, and
+  requests five (5) hours of time off, the request would be for three (3) hours since the two (2)
+  additionally worked hours are used first, and deducted from the request.
+- :guilabel:`Allow To Join Supporting Document`: enable this option for the employee to attach
+  documents to the time off request. This is useful in situations where documentation is required,
+  such as long-term medical leave.
+- :guilabel:`Kind of Leave`: select from the drop-down menu the type of leave this time off type is,
+  either :guilabel:`Time Off` or :guilabel:`Other`.
+- :guilabel:`Company`: if multiple companies are created in the database, and this time off type
+  only applies to one company, select the company from the drop-down menu. If this field is left
+  blank, the time off type applies to all companies.
+
+Allocation Requests section
+***************************
+
+- :guilabel:`Requires allocation`: if the time off must be allocated to employees, select
+  :guilabel:`Yes`. If the time off can be requested without time off being previously allocated,
+  select :guilabel:`No Limit`. if :guilabel:`No Limit` is selected, the following options do not
+  appear on the form.
+- :guilabel:`Employee Requests`: select :guilabel:`Extra Days Requests Allowed` if the employee is
+  able to request more time off than was allocated. For example, if ten (10) days are allocated to
+  the employee for this particular type of time off, and this option is enabled, the employee may
+  submit a request for more than ten (10) days. If employees should **not** be able to make
+  requests for more time off than what was allocated, select the :guilabel:`Not Allowed` option.
+- :guilabel:`Approval`: select the type of approval(s) required for the allocation of this
+  particular type of time off. :guilabel:`No validation needed` indicates that no approvals are
+  required. :guilabel:`Approved by Time Off Officer` indicates the :ref:`Time Off Officer
+  <time_off/time-off-officer>` set on this form must approve the allocation. :guilabel:`Set by Time
+  Off Officer` indicates that the :ref:`Time Off Officer <time_off/time-off-officer>` set on this
+  form must allocate the time off.
+
+Payroll section
+***************
+
+- :guilabel:`Work Entry Type`: if this time off type should create
+  :doc:`../hr/payroll/work_entries` in the *Payroll* application, select the type of work entry
+  created from the drop-down list.
+
```

Comment:
the work entries doc explains all of this thoroughly, which is why I linked it in here. I think that is sufficient, since it's based on the work entries created in payroll.